scripts/config/templates/nginx.conf.erb

```diff
@@ -111,6 +111,10 @@ http {
 
   rewrite ^/embed/(.*)$ /$1.html last;
 
+  location = "/search-q/storing" {
+    return 301 https://www.neighbor.com/search-q;
```

Comment:
Why hardcode neighbor.com?

Reply:
Oh good catch, I just pulled this from the only other static 301 redirects, which were for headless cms pages that don't have staging. Fixed